config/environments/production.rb

```diff
@@ -76,4 +76,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.paperclip_defaults = {
+      storage: :s3,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.